gui/styles/_two_panes.styl

```diff
@@ -234,7 +234,7 @@
     background-size cover
     background-position center center
 
-    for mimetype in audio binary code file folder image pdf presentation spreadsheet text video archive cozy-note
+    for mimetype in archive audio binary code contact cozy-note file folder image link pdf presentation spreadsheet text video
         &.file-type-{mimetype}
             background-image url("./images/type-icons/icon-type-" + mimetype + ".svg")
 
```

Comment:
Openly, we should mutualize the file list between drive web and desktop... First step can be fro Desktop to use the SVG sprite of Cozy-UI. Like that, we would not have to add every new icons to desktop since we've them all in UI...

Reply:
Wouldn't the first step be to upgrade `cozy-ui` then? It's still probably a lot of work but this could be done after upgrading `electron`.
I found the same list of icons we have in Desktop in the Drive Web repo. Are those left overs?

Reply:
> I found the same list of icons we have in Desktop in the Drive Web repo. Are those left overs?

Unfortunately, we still use them at one place we need to remove ;). 

Right for the UI upgrade. We should add this task somewhere in your roadmap ;)